src/Core/src/Hosting/Internal/MauiServiceProviderFactory.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.Maui.Hosting.Internal
 {
-	internal class MauiServiceProviderFactory : IServiceProviderFactory<IServiceCollection>
+	class MauiServiceProviderFactory : IServiceProviderFactory<IServiceCollection>
```

Comment:
Same comment as above, if it's on the namespace and folder it should be internal, or else move it out.

Reply:
This is just removing the keyword as internal is default.